frontend/packages/text-editor/src/TextRow.tsx

```diff
@@ -97,6 +97,7 @@ export const TextRow = ({
       </div>
       <div className={classes.rightCol}>
         <Button
+          aria-label="Slett"
```

Comment:
This file already has hard-coded values and language is not provided and useText is not available. Because useText-hook is within the UX-editor package and not the text-editor package. Sounds like we need to move the useText to shared, but that should be done within a separated PR.